packages/grafana-data/src/types/datasource.ts

```diff
@@ -547,6 +547,7 @@ export interface DataQueryRequest<TQuery extends DataQuery = DataQuery> {
 
   cacheTimeout?: string | null;
   queryCachingTTL?: number | null;
+  skipQueryCache?: boolean;
```

Comment:
Currently, CloudWatch Logs queries execute in an asynchronous manner. The first query starts a query and gets a query id back from AWS. Subsequent queries, use this query id to poll AWS for results.

Because of this, we currently implement our own `query` method for the CloudWatch data source because we need to be able to pass in headers to bypass the query cache so the polling requests don't loop infinitely until the cache expires.

Adding `skipQueryCache` will allow requests to have the `X-Cache-Skip` header set to bypass the cache.